createdir command on windows

Using `./zbox createdir --allocation $alloc --dirname "/roor/nested"` followed by `list-all` on Ubuntu shows expected output:

```
[{"name":"root","path":"/root","type":"d","size":0,"hash":""},{"name":"nested","path":"/root/nested","type":"d","size":0,"hash":""}]
```

But same command on windows followed by `list-all` shows this output (only one directory as opposed to two directories, one nested inside the other):
```
[{"name":"\\roor\\nested","path":"/\\roor\\nested","type":"d","size":0,"hash":""}]
```

> `[{"name":"\\roor\\nested","path":"/\\roor\\nested","type":"d","size":0,"hash":""}]`
> 
> roor ??

@lpoli it could be any user input name. 


The error has changed now, but issue still exists:

On Ubuntu this command: 
`./zbox createdir --allocation 67791a5a675e945ce026c061b3294a1287f62869f65d6ac13f2cc20a173862ab --dirname "/root/nested"` shows "/root/nested directory created".

On Windows, the same command:
`./zbox createdir --allocation 67791a5a675e945ce026c061b3294a1287f62869f65d6ac13f2cc20a173862ab --dirname "/root/nested"` shows:

```
0box-sdk        [ERROR]  2022/05/08 17:13:27.571089 dirworker.go:93: https://dev.0chain.net/blobber01Response: {"code":"invalid_path","error":"invalid_path: \\root\\nested is not absolute path"}
0box-sdk        [ERROR]  2022/05/08 17:13:27.571740 dirworker.go:39: {"code":"invalid_path","error":"invalid_path: \\root\\nested is not absolute path"}
0box-sdk        [ERROR]  2022/05/08 17:13:27.906549 dirworker.go:93: https://dev.0chain.net/blobber05Response: {"code":"invalid_path","error":"invalid_path: \\root\\nested is not absolute path"}
0box-sdk        [ERROR]  2022/05/08 17:13:27.909690 dirworker.go:39: {"code":"invalid_path","error":"invalid_path: \\root\\nested is not absolute path"}
0box-sdk        [ERROR]  2022/05/08 17:13:28.120789 dirworker.go:93: https://dev.0chain.net/blobber04Response: {"code":"invalid_path","error":"invalid_path: \\root\\nested is not absolute path"}
0box-sdk        [ERROR]  2022/05/08 17:13:28.125201 dirworker.go:39: {"code":"invalid_path","error":"invalid_path: \\root\\nested is not absolute path"}
0box-sdk        [ERROR]  2022/05/08 17:13:28.126321 dirworker.go:93: https://dev.0chain.net/blobber03Response: {"code":"invalid_path","error":"invalid_path: \\root\\nested is not absolute path"}
0box-sdk        [ERROR]  2022/05/08 17:13:28.126899 dirworker.go:39: {"code":"invalid_path","error":"invalid_path: \\root\\nested is not absolute path"}
```

@cnlangzi it still happens in `powershell.exe too`.

```
[ERROR]  2022/06/05 15:59:49.769142 dirworker.go:93: https://dev-1.devnet-0chain.net/blobber03Response: {"code":"invalid_path","error":"invalid_path: 
\\dir1\\dir2 is not absolute path"}
0box-sdk        [ERROR]  2022/06/05 15:59:49.773713 dirworker.go:39: {"code":"invalid_path","error":"invalid_path: \\dir1\\dir2 is not absolute path"}
0box-sdk        [ERROR]  2022/06/05 15:59:50.089129 dirworker.go:93: https://dev-1.devnet-0chain.net/blobber06Response: {"code":"invalid_path","error":"invalid_path: 
\\dir1\\dir2 is not absolute path"}
0box-sdk        [ERROR]  2022/06/05 15:59:50.092076 dirworker.go:39: {"code":"invalid_path","error":"invalid_path: \\dir1\\dir2 is not absolute path"}
```

--- a/zboxcore/sdk/dirworker.go
+++ b/zboxcore/sdk/dirworker.go
@@ -62,7 +62,7 @@ func (req *DirRequest) createDirInBlobber(blobber *blockchain.StorageNode) error
 	formWriter := multipart.NewWriter(body)
 	formWriter.WriteField("connection_id", req.connectionID)
 
-	dirPath := filepath.Join("/", filepath.ToSlash(req.name))
+	dirPath := filepath.ToSlash(filepath.Join("/", req.name))
 	formWriter.WriteField("dir_path", dirPath)
 
 	formWriter.Close()

